src/applications/personalization/profile/tests/e2e/address-validation/bad-unit.cypress.spec.js

```diff
@@ -0,0 +1,45 @@
+import { setUp } from 'applications/personalization/profile/tests/e2e/address-validation/setup';
+
+describe('Personal and contact information', () => {
+  describe('when entering an address with a bad unit', () => {
+    it('should successfully update on Desktop', () => {
+      setUp('bad-unit');
+
+      cy.get('#root_addressLine1')
+        .clear()
+        .type('225 irving st');
+      cy.get('#root_addressLine2')
+        .clear()
+        .type('Unit A');
+
+      cy.findByLabelText(/City/i)
+        .clear()
+        .type('San Francisco');
+      cy.get('#root_stateCode').select('CA');
```

Comment:
For whatever reason, it finds more than one 'State' label - so I won't be using `findByLabelText('State')` here